README.md

```diff
@@ -135,6 +135,49 @@ And then you chain everything together and it will just work :)
 UserQuery.new.where.not(name: nil).active_last_week.not_deleted.order(:id)
 ```
 
+## Generators
```

Comment:
the new heading are missing in the TOC https://github.com/rootstrap/arqo/pull/16/files#diff-04c6e90faac2675aa89e2176d2eec7d8R9